test_data/asm/block_to_block_with_bus.asm

```diff
@@ -22,7 +22,9 @@ machine Arith with
 
     let used = std::array::sum(sel);
 
-    bus_receive(ARITH_INTERACTION_ID, [0, x, y, z], latch * used, latch);
+    col witness bus_selector;
+    std::utils::force_bool(bus_selector);
+    bus_receive(ARITH_INTERACTION_ID, [0, x, y, z], latch * bus_selector, latch * bus_selector);
```

Comment:
This is an interesting one. In this test, we use a normal link and a bus interaction in parallel. Bus interactions used to be ignored, but now we actually do each call twice. That's fine, but then we need a new selector, because otherwise it makes the permutation argument invalid.